### DISCUSSION TITLE
Default Wallet Selection in GUI/CLI

### DISCUSSION THREAD

**Discussion by Meatball13** [Chia-Network/chia-blockchain#6976] [Ideas]
Could you please set up a flag or checkbox or something that would allow a user to set a 'default wallet' to start with the GUI/UI node so you can set it to autostart synching and mining without waiting for user input to select the wallet?

**Comment by loppefaaret** [comment]
does it not farm in the background of the key selection screen ? an active selected wallet is not needed for farming

**Reply by Meatball13** [reply]
Hmm, interesting.  I just tested it on Windows with the GUI using the 'chia farm summary' command and it does look like it starts farming before you select the key.  I only have one key, so not sure how it handles deciding what to farm if there's more than one key.

I guess this one can be set to answered and closed. :)

**Reply by loppefaaret** [reply]
selecting a key in the GUI, activates that wallet, and syncs it, and keeps it synced.
farming doesn't rely on an active wallet being selected
